src/components/checkbox/CheckboxLabel.tsx

```diff
@@ -65,7 +66,16 @@ export const CheckboxLabel = ({
       // inheritance on Android
       needsOffscreenAlphaCompositing={true}
     >
-      <View style={[IOStyles.row, { alignItems: "center", width: "100%" }]}>
+      <View
+        style={[
+          IOStyles.row,
+          {
+            alignItems: "center",
+            width: "100%",
+            columnGap: 8 * dynamicFontScale * spacingScaleMultiplier
```

Comment:
Maybe we can define a constant for this `8` multiplier

Reply:
@LeleDallas Thanks. I've also replaced the whole block with a HStack with the appropriate style to get the same behaviour. → [`312a614` (#348)](https://github.com/pagopa/io-app-design-system/pull/348/commits/312a6144449db214274151d7c67d1383a83f5bfe)